libs/ml/examples/word2vec_SGNS_dataloader/main.cpp

```diff
@@ -157,6 +157,8 @@ std::string ReadFile(std::string const &path)
 
 struct TrainingParams
 {
+  // TODO (#1585) something is broken here. if u set max_word_count to sth smaller like 10000, there
+  // would be an error at the end of the sentence
```

Comment:
It's a minor comment, but txt spk in comments makes me wince!

Reply:
ahh ur

Reply:
ah you are right, I should definitely make it more formal.